### DISCUSSION TITLE
`pkg list-installed --manual`

### DISCUSSION THREAD

**Discussion by Rudxain** [termux/termux-packages#17899] [Ideas]
Please add a simple API to list manually-installed packs, if possible

**Comment by sylirre** [comment]
`apt-mark showmanual`

Pay attention that bootstrap packages are always shown as manually installed. You will have a slightly larger list of manually installed packages than expected.

**Reply by Rudxain** [reply]
Thank you for the info! I tried searching for this everywhere, but found stuff that only worked for Debian-based distros
